tests/schemas/feature.json

```diff
@@ -1,10 +1,15 @@
 {
     "type": "object",
     "properties": {
-        "state": {"type": "string"},
+        "state": {"enum": ["enabled", "disabled", "beta"]},
         "exceptions": {
             "type": "array",
             "items": {"type": "object"}
+        },
+        "settings": {
```

Comment:
if we specify `settings` as optional field then we can say `additionalProperties: false`